config/vitest.config.browser.mts

```diff
@@ -22,6 +22,7 @@ const config = defineConfig({
   ],
   optimizeDeps: {
     exclude: ['kzg-wasm'],
+    include: ['vite-plugin-node-polyfills/shims/buffer', 'vite-plugin-node-polyfills/shims/global', 'vite-plugin-node-polyfills/shims/process'] 
```

Comment:
At first look, these additions seem to go against our as-pure-as-possible-browser-support direction strategy. 🤔 

Is this something to further investigate (why needed, what changed, where to adjust)?

(I just created a non-minifed VM (so not: EVM) bundle as a first test and searched for "Buffer" but did not find anything suspicious)

Reply:
I would agree that this was my surface thought as well.  We have these various shims by default as a part of the `vite-plugin-node-polyfills` dep for running our tests and vitest was having trouble reloading them or something (saw these messages about optimizing these deps over and over again in the logs) and the suggested fix was to add them to the `optimizeDeps` list in our config.  Given the effort on getting rid of polyfills, we might be able to just get rid of that plugin though I'm not 100% sure.  Worth researching.